### PR TITLE
fixes wrong days with tracing info on home screen

### LINKED ISSUE #853: Number of Active Days stuck
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug

Since 3 days in a row, the display of active days in the app shows "13/14 days". I saw the other issues related to a rounding issue of active days, but I think this is different, because in the meanwhile, the active days should be more than 14.

## Steps to reproduce the issue

I installed the app on the day when it came out. Also the updates were installed immediately on the days when the were released. On the day of the latest update 1.0.4 the days showed 13/14 before and after the update, and did not change since then.

## Technical details

- iOS Version: 13.5.1
- Device: iPhone Xs

### PR DESCRIPTION
**Pull request by ChristianKienle**
Closes #853

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/Risk/Risk.swift [hunk 1]
```diff
@@ -29,8 +29,9 @@ extension Risk {
 	struct Details {
 		var daysSinceLastExposure: Int?
 		var numberOfExposures: Int?
-		var numberOfHoursWithActiveTracing: Int
-		var numberOfDaysWithActiveTracing: Int { numberOfHoursWithActiveTracing / 24 }
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Services/Risk/Risk.swift:33]: This was the redundant calculation that is now gone.

File: src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+//
+
+import Foundation
+
+private extension TimeInterval {
+	static let SEC_PER_HOUR: TimeInterval = 3600.0
+	static let SEC_PER_DAY = SEC_PER_HOUR * 24.0
+}
+
+struct ActiveTracing {
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift:27]: This is the new structure holding the active tracing information.

File: src/xcode/ENA/ENA/Source/Workers/__tests__/ActiveTracingTests.swift [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import XCTest
+@testable import ENA
+
+final class ActiveTracingTests: XCTestCase {
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Workers/__tests__/ActiveTracingTests.swift:23]: Tests

**Review by SebastianWild — APPROVED** [review]

**Comment by inf2381** [comment]
Fixes #853

**Review by inf2381 — APPROVED** [review]

**Pull request merged**